frontend/components/Filter/OrganiserFilterDialog.tsx

```
@@ -189,23 +155,19 @@ export default function OrganiserFilterDialog({
-        <div className="mb-4">
-          <label className="text-sm font-medium text-gray-700 mb-1 flex items-center">
+      <div className="p-3 bg-gray-100 border border-gray-300 rounded-lg">
+        <h2 className="text-base text-center font-semibold mb-3">Filter Events</h2>
+        <div className="mb-3">
+          <label className="text-xs font-medium text-gray-700 mb-1 flex items-center">
             Sort By
             <span className="cursor-pointer ml-auto" onClick={toggleShowSortBy}>
-              {showSortBy ? <ChevronUpIcon className="h-6 w-6" /> : <ChevronDownIcon className="h-6 w-6" />}
+              {showSortBy ? <ChevronUpIcon className="h-5 w-5" /> : <ChevronDownIcon className="h-5 w-5" />}
             </span>
           </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)